src/classes/BDI_SettingsUI_CTRL.cls

```diff
@@ -91,8 +91,8 @@ public with sharing class BDI_SettingsUI_CTRL {
                 listSODonationFields = new list<SelectOption>();
                 
                 set<String> setFields = new set<String>();
-                setFields.addAll(BDI_Donations.dataImportFieldToOpportunityField.keySet());
-                setFields.addAll(BDI_Donations.dataImportFieldToPaymentField.keySet());
+                //setFields.addAll(BDI_Donations.dataImportFieldToOpportunityField.keySet());
```

Comment:
Need to do something here, but this dependence on these field lists needs to shift with our new scheme I think anyway